client/packages/host/package.json

```diff
@@ -23,7 +23,7 @@
     "start": "webpack-cli serve",
     "start-remote": "webpack-cli serve --env API_HOST='https://demo-open.msupply.org'",
     "start-local": "webpack-cli serve --env API_HOST='http://localhost:8000'",
-    "build": "yarn clean && webpack --env production",
+    "build": "webpack --env production",
```

Comment:
I don't want to remove clean - the build files really should be removed prior to building in order to remove artefacts.
Can you instead change the command:

```
    "clean": "rimraf ./dist/*",
```

tested - and works on windows and macOS 😉

Reply:
https://webpack.js.org/guides/output-management/#cleaning-up-the-dist-folder

https://github.com/openmsupply/open-msupply/blob/2fac9ff941c46e9b50f9bf67ec4ceb0fa1608dab/client/packages/host/webpack.config.js#L43-L45

dist folder is cleaned without `clean` script, webpack does this now. I tried by putting random files in there between builds, and they are removed.

If we remove `./dist` folder, build files will show in git, since i commented out (in favour of using .gitingore file within dist folder):

https://github.com/openmsupply/open-msupply/blob/2fac9ff941c46e9b50f9bf67ec4ceb0fa1608dab/client/.gitignore#L12

If that dist folder does not exist in repo when it's initially pulled, remote server fails to compile (try a new clone)